Rerender feedstock

saraedum opened the pull request:
to fix #9

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

saraedum commented:
@conda-forge-admin please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.17.5, conda-smithy 3.2.9, and conda-forge-pinning 2019.01.16

saraedum committed:
Bump build number

saraedum committed:
Rebuild configure file

to fix `/home/conda/feedstock_root/build_artifacts/gf2x_1547744508416/work/config/missing: line 81: aclocal-1.15: command not found`

isuruf committed:
Update build.sh

saraedum commented:
Closing some very old PRs.
